backend/src/routes/api/modelRegistryRoleBindings/index.ts

```diff
@@ -0,0 +1,72 @@
+import { FastifyReply, FastifyRequest } from 'fastify';
+import { secureAdminRoute } from '../../../utils/route-security';
+import { KubeFastifyInstance } from '../../../types';
+import {
+  createModelRegistryRoleBinding,
+  deleteModelRegistriesRolebinding,
+  listModelRegistryRoleBindings,
+} from './modelRegistryRolebindingsUtils';
+import { V1RoleBinding } from '@kubernetes/client-node';
+import { getModelRegistryNamespace } from '../modelRegistries/modelRegistryUtils';
+
+export default async (fastify: KubeFastifyInstance): Promise<void> => {
+  const mrNamespace = await getModelRegistryNamespace(fastify);
```

Comment:
@gitdallas sorry I didn't catch this when you first rebased - `getModelRegistryNamespace` changed, it's no longer async and instead of calling it at the top of this exported function it needs to be called within the `try` inside each of the fastify callbacks below (right before `mrNamespace` is used in each one). See how it is used now in `backend/src/routes/api/modelRegistries/index.ts`.

Reply:
Too many moving targets 😆